### DISCUSSION TITLE
Does Jellyfin-SSO supports PKCE ?

### DISCUSSION THREAD

**Discussion by LeVraiRoiDHyrule** [9p4/jellyfin-plugin-sso#187] [Q&A] [answered]
Hi,
I am looking to use Jellyfin SSO with Zitadel. By defaults, Zitadel (as well as most OpenID providers now) chooses PKCE for connection with a client, instead of asking for ClientID + Client Secret. Does Jellyfin SSO supports it ?
Thanks in advance for any answer.

**Comment by 9p4** [comment]
The plugin doesn't support it yet.

**Reply by LeVraiRoiDHyrule** [reply, marked as answer]
Hi, I am answering months later but I've just tried with PKCE (with Zitadel as my provider) and it worked ! So I don't know if that was intended or not but PKCE works now. A client secret still needs to be set, but it won't be taken into account. That's great ! I don't know if it will work with all PKCE providers but maybe it could be worth mentionning so people know it is compatible.
